airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -449,7 +449,7 @@ class SnowflakeSqlApiOperator(SQLExecuteQueryOperator):
             When executing the statement, Snowflake replaces placeholders (? and :name) in
             the statement with these specified values.
     :param deferrable: Run operator in the deferrable mode.
-    """  # noqa
+    """  # noqa: D205, D400
```

Comment:
I guess originally it complains about too long line in docstring, however not it doesn't violent if formatters / linter but we hide accidentally:
- `D205`	[blank-line-after-summary](https://docs.astral.sh/ruff/rules/blank-line-after-summary/)
- `D400`	[ends-in-period](https://docs.astral.sh/ruff/rules/ends-in-period/)